tests/conftest.py

```diff
@@ -0,0 +1,6 @@
+import pytest
+
+@pytest.fixture(scope='function')
+def mock_data():
```

Comment:
Once you start coding and writing tests, make sure to delete these example tests and fixtures.